### DISCUSSION TITLE
Automate periodic Sync on re-plays to help fix out of sync issues

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#179] [Feature requests]
When I'm pushing the limits of my system, I will find that streams on replay can get out of sync. So to address this, on the main feed, e.g F1 Live, I'll press the s key periodically. It may be beneficial, to have option to set this stream to be able to initiate this automatically, e.g. every 60 seconds.  So potentially a right click option to enable this on the stream that the user selects as the lead stream.

**Comment by Macaw1975** [comment]
Only issue with this is that it would solidify any streams that go out of sync from the broadcast side of things. There was a race I watched were Perez's OBC was suddenly 4 seconds during the middle of the race even after spamming S. So when I gave my periodic S input, I'd need to nudge him back into alignment manually. An automated Periodic Sync would unfortunately solidify issues like that one.
